lib/syskit/scripts/log_runtime_archive.rb

```diff
@@ -18,11 +18,20 @@ def self.exit_on_failure?
            type: :numeric, default: 600, desc: "polling period in seconds"
     option :max_size,
            type: :numeric, default: 10_000, desc: "max log size in MB"
+    option :FREE_SPACE_LOW_LIMIT,
```

Comment:
command line arguments are lowercase. Using the constant was for the 'default' value which is currently not guaranteed to be equal to the class defaults.